Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php

```diff
@@ -272,4 +272,35 @@ public function testWrongCasedPhpdocTagIsNotAltered()
 EOF;
         $this->makeTest($expected);
     }
+
+    public function testInlineDoc()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * Does stuffs with stuffs.
+     *
+     * @param array $stuffs {
+     *     @type bool $foo
+     *     @type int $bar
```

Comment:
it just got curious about it, that is all, now I know it will be done by the other fixer, so

>Can we leave that to another PR,

sure :) 
thanks for clearing it up :+1: